.github/workflows/ci.yml

```diff
@@ -187,6 +193,11 @@ jobs:
     #     path: ${{ env.SCCACHE_PATH }}
     #     key: sccache-${{ runner.os }}-${{ hashFiles('pnpm-lock.yaml', 'patches/**') }}
 
+    # Needed because electron-builder's bundled FPM doesn't support arm64
+    - name: Install FPM (for packaging)
+      if: ${{ matrix.os == 'ubuntu-22.04-arm64-4-cores' }}
+      run: sudo gem install fpm
```

Comment:
Starting with `electron-builder` [26.0.18](https://github.com/electron-userland/electron-builder/releases/tag/v26.0.18), this workaround will no longer be needed as Linux arm64 binaries have been added for `fpm`.

However, that version is currently in pre-release, so I guess folks will want to wait for a next official release prior to updating to that version.